specifyweb/frontend/js_src/lib/components/FormMeta/index.tsx

```diff
@@ -161,7 +161,7 @@ function MetaDialog({
                 <EditHistory resource={resource} />
               </ProtectedAction>
             </ProtectedTool>
-            {isTreeResource(resource) && (
+            {isTreeResource(resource) && resource.id !== undefined && (
```

Comment:
Could we change `resource.id !== undefined` to `!resource.isNew()`? 
This would stay consistent with the rest of the code base and I personally think it is more explicit. 

To a new developer, or even someone going through the code `resource.isNew()` leaves no doubt as to _why_ its there. It might not be immediately clear that we are hiding the QueryTreeUsages component when the resource is new, or they might not have realized that an undefined id means a new resource. 
Alternatively, `isNew()` is a function from Backbone. If/when we decide to get rid of Backbone, it would be much easier to detect and thus replace this line.